Userland/Utilities/lsusb.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, Jesse Buhagiar <jooster669@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteBuffer.h>
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <AK/LexicalPath.h>
+#include <AK/String.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/DirIterator.h>
+#include <LibCore/File.h>
+#include <stdio.h>
+#include <unistd.h>
+
+static const char* format = "Device {}: ID {:04x}:{:04x}";
```

Comment:
Please don't do this, I'm pretty sure this circumvents the compiletime format string checker.
it's also used only once, so this is pretty pointless :P